jekyll/_cci2/resource-class-overview.adoc

```diff
@@ -0,0 +1,110 @@
+---
+contentTags: 
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+  - Server v2.x
+---
+= Resource class overview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: An overview of resource classes in CircleCI
+:icons: font
+:toc: macro
+:toc-title:
+
+Use resource classes to:
+
+* Specify compute resource size (CPU and RAM) for running a link:/docs/concepts/#jobs[job].
+* Optimize executors based on historical trends with resource class Insights.
+* Access the link:/docs/using-arm[Arm] execution environment.
```

Comment:
why is ARM called out here?

Reply:
@sebastian-lerner because it's the only execution env that is specified just using the resource class key. I agree the way I've phrased it is weird but that just seems to be how it is?: 

Linux - machine executor - linux image
Docker - Docker executor - docker image
Windows - machine executor - windows image
GPU - machine executor - linux or windows gpu image
Arm - machine executor - Linux image - Arm resource class
macOS - macOS executor - Xcode version

Reply:
got it.  still feels weird to include in this list...if you wanna leave it though i'm not totally opposed

Reply:
that could work